spec/services/github_org_membership_service_spec.rb

```diff
@@ -55,4 +65,63 @@ def build(*_args)
       end
     end
   end
+
+  describe '#ex_members_ids' do
+    context "when membership doesn't exist" do
+      before do
+        allow(client).to receive(:org_members).with(organization.login)
+                                              .and_return(github_response_2, github_response)
+        service.import_all_from_organization(organization)
+      end
+
+      it 'returns an array with ex_member id' do
+        expect(organization.members.where(id: service.ex_members_ids(organization)).first.gh_id)
+          .to eq(20)
+      end
+    end
+
+    context 'when membership exists' do
+      before do
+        allow(client).to receive(:org_members).with(organization.login).and_return(github_response)
+        service.import_all_from_organization(organization)
+      end
+
+      it 'returns an array with ex_member id' do
+        expect(service.ex_members_ids(organization)).to contain_exactly
```

Comment:
No te falta especificar qué cosa quieres que contenga exactamente? Entiendo que `contain_exactly` recibe como argumento los elementos que quieres.

Reply:
Ahora si, debiera decir `.to be_empty`